README.md

```diff
@@ -85,6 +87,7 @@ mkdir build
 cd build
 cmake ..
 make
+ln -s ../data 
```

Comment:
You've removed your explanation. I'd prefer it to be re-instated to explain why the data directory is there. Perhaps we should have this line in its own code box below the explanation, too.